packages/dds/tree/src/test/simple-tree/api/schemaFactory.spec.ts

```diff
@@ -738,6 +747,15 @@ describe("schemaFactory", () => {
 			const factory = new SchemaFactoryAlpha("test");
 			class NamedRecord extends factory.record("name", factory.number) {}
 			const namedInstance = new NamedRecord({ x: 5 });
+			const x: number = namedInstance.x;
+			// TODO: this should not compile as the typing is incorrect (y is undefined, not number)
+			const y: number = namedInstance.y;
+			delete namedInstance.x;
+
+			// TODO: this should throw a nicer error, which should be validated here.
+			assert.throws(() => {
+				delete namedInstance.z;
+			});
```

Comment:
This highlights some new known issues.

Reply:
I removed the second issue here, since you fixed it. Did you want to file an item for the first?